packages/react-native-bpk-component-button/src/__snapshots__/BpkButton-test.android.js.snap

```diff
@@ -4,16 +4,11 @@ exports[`Android API levels Android API<21 should render without ripple 1`] = `
 <View
   style={
     Array [
-      Array [
-        Object {
-          "borderRadius": 40,
-          "overflow": "hidden",
-        },
-      ],
-      null,
       Object {
-        "backgroundColor": "rgb(0, 215, 117)",
```

Comment:
Any reason backgroundColor is not set anymore?

Reply:
It doesn't seem to be necessary, I tried removing it from the outer container and everything works fine